lib/DateTime/TimeZone.pm

```diff
@@ -687,14 +696,18 @@ for the given datetime.  This takes into account historical time zone
 information, as well as Daylight Saving Time.  The offset is
 determined by looking at the object's UTC Rata Die days and seconds.
 
-=head2 $tz->offset_for_local_datetime( $dt )
+=head2 $tz->offset_for_local_datetime( $dt, [ $ignore_missing_spans ] )
 
 Given a C<DateTime> object, this method returns the offset in seconds
 for the given datetime.  Unlike the previous method, this method uses
 the local time's Rata Die days and seconds.  This should only be done
 when the corresponding UTC time is not yet known, because local times
 can be ambiguous due to Daylight Saving Time rules.
 
+If C<$ignore_missing_spans> is true and the local time for C<$dt> does not
```

Comment:
I wonder if it makes sense to _document_ this talking about "spans". Note that the docs do not mention this term at all, because it's really just an internal detail.

I think this might be better presented as `$fall_back_on_skipped_time` or something like that. Saying "the next span up" seems wrong too. Aren't we going to the _previous_ span? But anyway, I'd say something like this:

> If `$fall_back_on_skipped_time` is true and the local time for C<$dt> does not exist in the time zone (typically because of a DSST change), then we will fall back to nearest previous offset for the local time.

Reply:
Hmm, we're going to the previous span but that moves the time forward, so I'd add to the above ...

> Note that moving to a previous offset will typically move the time *forward*, since the new offset from UTC will be larger.